OpenRA.Game/Graphics/Animation.cs

```diff
@@ -147,6 +148,30 @@ public bool ReplaceAnim(string sequenceName)
 			return true;
 		}
 
+		public void PlayNTimesThen(string sequenceName, int times, Action after)
```

Comment:
Can we please call this `public void PlayMultipleThen(string sequenceName, int repeats, Action after)`?